module/VuFind/src/VuFind/ILS/Driver/Alma.php

```diff
@@ -1815,9 +1810,26 @@ protected function getStatusesForInventoryTypes($ids, $types)
                 // Physical
                 $physicalItems = $marc->getFields('AVA');
                 foreach ($physicalItems as $field) {
-                    $avail = $marc->getSubfield($field, 'e');
+                    $available = null;
+                    $statusText = '';
+                    if ($this->locationTypeToItemStatus) {
+                        $locationCode = $marc->getSubfield($field, 'j');
+                        $library = $marc->getSubfield($field, 'b');
+                        [$available, $statusText] = $this->getItemStatusFromLocationTypeMap(
+                            $this->getLocationType($library, $locationCode)
+                        );
+                    }
+
+                    if (null === $available) {
+                        $availStr = strtolower($marc->getSubfield($field, 'e'));
+                        $available = 'available' === $availStr;
+                        // No status message available, so set it based on availability:
+                        $statusText = $available ? 'Item in place' : 'Item not in place';
```

Comment:
Is "Item in place" Alma terminology? If not, any particular reason this language was chosen?

Reply:
It is, and those are what Alma uses in the item statuses.